scipy/stats/tests/test_kdeoth.py

```diff
@@ -485,3 +485,14 @@ def test_seed_sub(gkde_trail):
     test_seed_sub(gkde_2d)
     gkde_2d_weighted = stats.gaussian_kde(xn_2d, weights=wn)
     test_seed_sub(gkde_2d_weighted)
+
+
+def test_gh10205():
+    # check that bug reported in gh-10205 (not positive definite) is resolved
+    np.random.seed(0)
+    mu = np.array([1, 10, 20])
+    sigma = np.array([[4, 10, 0], [10, 25, 0], [0, 0, 100]])
+    data = np.random.multivariate_normal(mu, sigma, 1000)
+    stats.gaussian_kde(data.T)
```

Comment:
This test should be compared to carefully created gold standard. You have added seeded the function, so there is a greater need to have reproducible runs. I suggest you compare within thresholds instead of absolutes.

Reply:
> This test should be compared to carefully created gold standard.

That is not the purpose of this test. This is a test that the bug report is resolved, so the example is from the issue. It trusts that other tests check the accuracy of the rest of the code.

> You have added seeded the function, so there is a greater need to have reproducible runs.

The seed is set so that behavior will be reproducible, yes. The current standard is for the test suite to be deterministic. Personally, I think a nondeterministic test suite that finds new bugs (and reports the information needed to reproduce them) might be nice, that is not what is currently done.

> I suggest you compare within thresholds instead of absolutes.

That is what I wrote [here](https://github.com/scipy/scipy/pull/13231/files#r541386485).